pkgs/tools/networking/sniffglue/default.nix

```diff
@@ -15,13 +22,17 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [ libpcap libseccomp ];
+  buildInputs = [
+    libpcap
+  ] ++ lib.meta.availableOn stdenv.hostPlatform libseccomp [
```

Comment:
```suggestion
  ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform libseccomp) [
```

Reply:
Done, but I think the claim of 
```
  ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform libseccomp) [
```
being better than
```
  ] ++ lib.optionals stdenv.isLinux [
```
or even
```
  ] ++ lib.optional stdenv.isLinux libseccomp;
```
is somewhat questionable.

Reply:
Indeed `libseccomp` (or just, seccomp) is only available on linux, but it's our human knowledge not some concrete truth. It can be broken on linux-musl, it can be broken on some specific architecture, it can be one day supported on bsd or even windows. `availableOn` better represents the actual condition we are checking with.

Reply:
The sandbox code in sniffglue is configured through `#[cfg(target_os = "linux")]`, if a BSD has `seccomp` in its pkg-config system it still won't be used and on Linux the build will fail if libseccomp is not available.